src/clj/witan/send/goal_seek.clj

```diff
@@ -148,19 +151,18 @@
                      :modify-transition-by)
         _ (println "Modifier:" modifier)
         projection (s/run-send-workflow config false)
-        base-year (+ 1 (apply max (into (sorted-set) (map :calendar-year (:transitions projection)))))
-        output (:send-output projection)
-        result (into {} (map #(assoc {} %1 %2) (range base-year (+ base-year (count output))) output))
-        target-year-result (get-in result [target-year :by-state])
-        target-pop-result (get-target-pop config target-year-result)]
+        result (tc/dataset (eduction stcio/simulated-transitions->transition-counts-xf (:simulated-transitions projection)))
+        census (stc/transition-counts->census-counts result (apply min (:calendar-year result)))
+        summary (summary/seven-number-summary census [:calendar-year :setting :need :academic-year] :transition-count)
+        target-pop-result (get-baseline-population (tc/rows summary :as-maps) target-year (dissoc m :modify-transition-by))]
```

Comment:
I think it is possible to write `get-baseline-population` as a tmd/tc thing that might run faster.